scripts/bootloader/provision.py

```diff
@@ -16,7 +16,9 @@
 def generate_provision_hex_file(s0_address, s1_address, hashes, provision_address, output):
     # Add addresses
     provision_data = struct.pack('III', s0_address, s1_address, len(hashes))
-    provision_data += b''.join(hashes)
+    for mhash in hashes:
```

Comment:
Why do we need this? (hehe)

Reply:
The implementation introduces a word in front of each PK hash which indicates whether it is valid or not.